Frontend/Gmfctn/src/app/core/services/users-service/user.service.ts

```diff
@@ -0,0 +1,77 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable, OnDestroy } from '@angular/core';
+import { BehaviorSubject, EMPTY, Observable, of } from 'rxjs';
+import { catchError, map, tap } from 'rxjs/operators';
+import { Achievement } from 'src/app/shared/models/achievement';
+import { UserSI } from 'src/app/shared/models/user-short-info';
```

Comment:
It's not that good practice to have a subset of the entity as a separate interface 
Better to do like this: `export type UserSI  = Pick<User, 'id' | 'firstName' | 'lastName' | 'xp' | 'avatarId'>`

Reply:
Fixed